backend/canisters/user_index/impl/src/updates/register_user.rs

```diff
@@ -176,14 +189,14 @@ fn commit(
         runtime_state.data.referral_codes.claim(code, user_id, now);
 
         runtime_state.queue_payment(PendingPayment {
-            amount: 50_000, // Approx $14
+            amount: 50, // Approx $14
```

Comment:
Should keep this at the correct value on prod

Reply:
Doh!